README.md

```diff
@@ -40,7 +40,23 @@ Edit your Sphinx's `conf.py`
     html_theme_path = pylons_sphinx_themes.get_html_themes_path()
     ```
 
-3.  If you were previously using the git submodule method to use the
+3.  (Recommended) Enable [Ethical
```

Comment:
This is actually pretty cool that it is off by default with the theme (you don't want to just enable it for consumers of your themes) but that it can be easily enabled. By changing the order here users of the theme can also position it where desired.

Your docs are already ahead of ours! I need to catch up.